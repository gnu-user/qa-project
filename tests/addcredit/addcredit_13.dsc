TITLE
    Non-Privileged Amount Input Validation (invalid input, negative value)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the addcredit 
    transaction. When prompted, enter a negative decimal input for the amount 
    of credit to add, and verify that it is not accepted and an error occurs.
DEPENDENCIES
    addcredit_10
