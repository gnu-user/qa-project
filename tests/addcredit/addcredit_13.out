Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the amount of credit to add: 
Invalid credit value, amount cannot be less than 0.00.